blaze/expr/functions.py

```diff
@@ -77,6 +80,27 @@ def sqrt(o):
          blazemath: Expr,
          reductions: Expr}
 
+binary_types = {
+    builtins: [(object, object)],
+    np: [((np.ndarray, np.number), (np.ndarray, np.number))],
+    pymath: [(Number, Number)],
+    blazemath: [(Expr, (Expr, base)), (base, Expr)]
+}
+
+
+binary_mappings = {
```

Comment:
Can we rename this `fallback_binary_mapping` or something to imply that this is not the default place to look for the binary function mappings.